communicationPlugIn-Kafka/src/main/java/com/cgi/bonnie/configuration/TopicConfig.java

```diff
@@ -40,14 +36,11 @@ public NewTopic bonnieMessageTopic() {
         return TopicBuilder.name(topicMessage).build();
     }
 
+    //fixme  there need only 2 topic -> one bean is not necessary. This property is already defined in KafkaMessageService class, but there <String,String>
     @Bean
     public KafkaTemplate senderProps(KafkaProperties kafkaProperties) {
         Map<String, Object> props = kafkaProperties.buildProducerProperties();
 
-        props.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class.getName());
-        props.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, JsonSerializer.class.getName());
-        props.put(JsonDeserializer.TRUSTED_PACKAGES, "*");
-
```

Comment:
This is fixed in the main branch. Rebase this branch on main because it is 1 commit behind